assets/js/common/Table/CollapsibleTableRow.jsx

```diff
@@ -1,51 +1,63 @@
-import React, { Fragment, useState } from 'react';
+import React, { useState } from 'react';
 import classNames from 'classnames';
 import { EOS_KEYBOARD_ARROW_DOWN } from 'eos-icons-react';
 
 function CollapsibleTableRow({
   columns,
   item,
   collapsibleDetailRenderer,
+  wrapCollapsedRowInCell = true,
   renderCells = () => {},
   colSpan = 1,
   className,
   collapsedRowClassName,
 }) {
   const [rowExpanded, toggleRow] = useState(false);
   const collapsibleRowSpan = collapsibleDetailRenderer ? colSpan + 1 : colSpan;
+  const collapsibleItem =
+    collapsibleDetailRenderer &&
+    collapsibleDetailRenderer(item, { columns, rowExpanded, renderCells });
+  const isCollapsible = !!collapsibleItem;
 
   return (
     <>
       <tr className={className}>
         {collapsibleDetailRenderer && (
           <td
-            className="pl-4 border-b border-gray-200 bg-white"
+            className={classNames(
+              'border-b border-gray-200 bg-white',
+              { 'pl-2': !isCollapsible },
+              { 'pl-4': isCollapsible }
+            )}
+            aria-label="collapsible-icon-cell"
             onClick={() => toggleRow(!rowExpanded)}
           >
-            <EOS_KEYBOARD_ARROW_DOWN
-              className={classNames(
-                'cursor-pointer self-center fill-gray-500',
-                {
-                  'transform rotate-180': rowExpanded,
-                }
-              )}
-            />
+            {isCollapsible && (
```

Comment:
In which cases the outer `collapsibleDetailRenderer` would be true, but `isCollapsible` false?
Meaning that we could have collapsible table rows without the icon 🤔

Reply:
I see, thanks!